docs/guides/contributing.md

```diff
@@ -50,7 +50,7 @@ The merge commit message should adhere to the [conventional commit guidelines](h
 ### Coding guidelines
 * If there is code that needs to be tested, there should be tests written for it.
 * If there are any additions or changes to the public API, the documentation should be updated. 
-* Files should be added to the appropriate folder to keep modules and objects within the correct scope.  
+* Files should be added to the appropriate folder to keep modules and objects within the correct scope.
```

Comment:
maybe add examples for each of these three points?

Reply:
This file should not have been changed =D The only reason it is listed here is because I added/removed some whitespace. That being said: I think we should look into your comment when we do the big overhaul of our documentation.